lib/importproject.cpp

```diff
@@ -879,6 +905,14 @@ bool ImportProject::importVcxproj(const std::string &filename, std::map<std::str
                     fs.defines += ";__AVX512__";
                 additionalIncludePaths += ';' + i.additionalIncludePaths;
             }
+            for (const ConfigurationPropertyGroup &c : configurationPropertyGroups) {
+                if (!c.conditionIsTrue(p))
+                    continue;
+                if (c.useUnicode) {
```

Comment:
I don't remember the details here but as far as I know we can check several configurations.

so if multiple c conditions are true then UNICODE define might be added more than once?

and if the first `c.useMfc` is true and then second `c.useMfc` is false then `fs.useMfc` will be false but UNICODE is added to the defines?

or is it only possible that only 1 `c.conditionIsTrue` will be true?

Reply:
Multiple conditions can be true. I will test with VS what msbuild would do.